ipasetup.py.in

```diff
@@ -1,5 +1,4 @@
-#!/usr/bin/python2
-# Copyright (C) 2014  Red Hat
+# Copyright (C) 2016  Red Hat
```

Comment:
The file was created 2016. I forgot to correct the year when I copied over the license header.